### PR TITLE
updated nl Connection_Errors.txt

### PR DESCRIPTION
**Pull request by TotallyFish**
# SCP:SL Translation PR Template

## Language
nl

## Type of change
- [ ] Added new language translation
- [ x] Changed a currently existing translation

## Have you tested the changes you've made?
- [x ] Yes
- [ ] No

# Checklist:
- [ x] I have made sure to not commit changes to other stuff than my own or the default english one
- [ ] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [x ] I have made sure that I'm not submitting a duplicate translation 
- [ x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

### CODE REVIEW

**Comment by ShingekiNoRex** [comment]
Hey we have some changes since you opened your pull request. Can you make sure they are up to date?
For details:
Fixes a missing translation for SCP-207 death
https://github.com/northwood-studios/SCPSL-Translations/commit/01925332a529304ba6a2374b939fc603c419c825

Added new connection errors
https://github.com/northwood-studios/SCPSL-Translations/commit/e7bc364df5b30d5544776d8cd8fdc34878eb0814

Added recontainment damage type
https://github.com/northwood-studios/SCPSL-Translations/commit/1295726cc1d3194f34afec8544994db03bbea2f3

049 can now reanimate regardless of cause of death
https://github.com/northwood-studios/SCPSL-Translations/commit/e0e94ef445e4cf54061a1f94e28771004822f39d

**Comment by ShingekiNoRex** [comment]
Please resolve the conflicts

File: Translations/nl/Connection_Errors.txt [hunk 1]
```diff
@@ -12,8 +12,8 @@
...
 <size=80>EEN OGENBLIK GEDULD</size>\n<size=35>Verbinding tot stand brengen...</size>
-<size=80>RONDE HERSTARTEN</size>\n<size=35>De server is het level aan het herstarten...</size>
+<size=80>RONDE HERSTARTEN</size>\n<size=35>De server is de ronde aan het herstarten...</size>
 <size=80>EEN OGENBLIK GEDULD</size>\n<size=35>Tutorial laden...</size>
 <size=80>VERSIE INCOMPATIBILITEIT</size>\nJouw spel versie is anders dan die van de server\n(Jouw - [your]; Server - [server])
-<size=80>CONNECTED</size>
-<size=35>Faciliteit aan het laden...</size>
\ No newline at end of file
+<size=80>VERBONDEN</size>\n<size=35>Het level is aan het laden...</size>
+<size=35>Faciliteit aan het laden...</size>
```
Comment (Dat-Joseph) [inline Translations/nl/Connection_Errors.txt]: Update please to latest standard.

File: Translations/nl/Death_Causes.txt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 Het is <b>[user]</b>'s lichaam, hij was een [class]!\n\nDoodsoorzaak: [cause]
-In stukken gescheurd door SCP-096.
-Powerful bite to cranium.
-Melted by strongly corrosive substance.
-Post-explosion wounds.
\ No newline at end of file
+Enorme snijwonden en overmatig veel bloodverlies.
+Grote, diepe bijt wonden rond de schedel zijn aanwezig.
+Versmolten door een enorm bijtend stof.
+Shrapnel en derdegraads brandwonden door een explosie.
```
Comment (Dat-Joseph) [inline Translations/nl/Death_Causes.txt]: Update please to latest standard.

**Comment by Dat-Joseph** [comment]
Closing this PR cause it is outdated, please open a new pull request for NL translation. 👍

**Pull request closed without merging**